db/migrate/20230614070723_bring_inventory_in_and_out_to_invetory_level.rb

```diff
@@ -0,0 +1,92 @@
+class BringInventoryInAndOutToInvetoryLevel < ActiveRecord::Migration[7.0]
+  # this data migration is for the perpose to bring inventory_in and inventory_out
+  # to current inventory level so that in-case of kit de-allocation we do not face "inconsistency inventory in" error
+  class AllocationService
```

Comment:
Is there a reason you copy/pasted this entire class into the migration?

Reply:
removed this migration as per discussion with @cielf